dhall/src/Dhall/Main.hs

```diff
@@ -198,6 +199,12 @@ parseMode =
               <>  Options.Applicative.help
                     "Output import dependency graph in dot format"
               )
+        <|>
+          Options.Applicative.flag' (Just ListImmediateDependencies)
+              (   Options.Applicative.long "immediate"
```

Comment:
How about calling this flag `--immediate-dependencies` and then renaming `--list` to `--transitive-dependencies`?

Reply:
... I was wondering what to call these.. :)